lerna.json

```diff
@@ -1,5 +1,5 @@
 {
-  "useNx": false,
+  "useNx": true,
```

Comment:
Reading this page, it all makes sense the order of operations being taken care of by lerna here with help from Nx to see it that it cannot build the helpers package before testids. https://nx.dev/recipes/adopting-nx/lerna-and-nx Makes sense :+1: